pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -30,6 +31,8 @@ const (
 	WindowsOSLabel = "node.openshift.io/os_id=Windows"
 	// WorkerLabel is the label that needs to be applied to the Windows node to make it worker node
 	WorkerLabel = "node-role.kubernetes.io/worker"
+	// VersionAnnotation indicates the version of WMCO that configured the node
+	VersionAnnotation = "windowsmachineconfiguration.openshift.io/config-version"
```

Comment:
I am not so sure if `windowmachineconfiguration` is the right choice. Is there a reason for `config-version` vs `version`? I prefer the shorter `version`. Plus there is some stuttering going on. How about `windowsmachineconfig.openshift.io/version`?